document/src/test/resources/config/application.yml

```diff
@@ -46,3 +46,8 @@ valtimo:
     versioning.enabled: true
     plugin:
         encryption-secret: "abcdefghijklmnop"
+
+logging:
+    level:
+        org.hibernate.SQL: DEBUG
+        org.hibernate.type: TRACE
```

Comment:
Do we want to keep this?
Seems obsessive for non-development purposes

Reply:
This is used in the integration-tests and is very useful for debugging issues with the document search